polyfills/common/webapi_poly_common.js

```diff
@@ -24,7 +25,10 @@
       console.log('*-*-* NavConnectHelper: ' + text);
     }
 
-    return new Promise((resolve, reject) => {
+    // It actually makesmore sense having this here...
```

Comment:
Not to do now, but to take into account: with this change we can remove _currentRequestId from all the other classes!